[Grid] Rename child/children references to gridItem/gridItems.

sammygill opened the pull request:
#### 3d47cc57612d6c856089aa3dfb6cdb9dd598e538
<pre>
[Grid] Rename child/children references to gridItem/gridItems.
<a href="https://bugs.webkit.org/show_bug.cgi?id=276978">https://bugs.webkit.org/show_bug.cgi?id=276978</a>
<a href="https://rdar.apple.com/problem/132364014">rdar://problem/132364014</a>

Reviewed by Alan Baradlay.

These renderers participate in grid layout and are grid items so
renaming them to such in the code makes it more consistent with the
spec and easier to read. There are some instances that are used as part
of generic render tree logic and not specifially grid layout which I
left alone.

There were also a couple of instances of &quot;childIterator&quot; in the
grid track sizing algorithm that I changed to &quot;subgridIterator,&quot;&quot; since
that&apos;s what they are.

* Source/WebCore/rendering/AncestorSubgridIterator.cpp:
(WebCore::AncestorSubgridIterator::operator++):
* Source/WebCore/rendering/Grid.cpp:
(WebCore::Grid::insert):
(WebCore::GridIterator::GridIterator):
(WebCore::GridIterator::nextGridItem):
(WebCore::GridIterator::isEmptyAreaEnough const):
(WebCore::GridIterator::createForSubgrid):
* Source/WebCore/rendering/Grid.h:
* Source/WebCore/rendering/GridBaselineAlignment.cpp:
(WebCore::GridBaselineAlignment::logicalAscentForGridItem const):
(WebCore::GridBaselineAlignment::ascentForGridItem const):
(WebCore::GridBaselineAlignment::descentForGridItem const):
(WebCore::GridBaselineAlignment::isDescentBaselineForGridItem const):
(WebCore::GridBaselineAlignment::isOrthogonalGridItemForBaseline const):
(WebCore::GridBaselineAlignment::isParallelToAlignmentAxisForGridItem const):
(WebCore::GridBaselineAlignment::baselineGroupForGridItem const):
(WebCore::GridBaselineAlignment::updateBaselineAlignmentContext):
(WebCore::GridBaselineAlignment::baselineOffsetForGridItem const):
(WebCore::GridBaselineAlignment::logicalAscentForChild const): Deleted.
(WebCore::GridBaselineAlignment::ascentForChild const): Deleted.
(WebCore::GridBaselineAlignment::descentForChild const): Deleted.
(WebCore::GridBaselineAlignment::isDescentBaselineForChild const): Deleted.
(WebCore::GridBaselineAlignment::isOrthogonalChildForBaseline const): Deleted.
(WebCore::GridBaselineAlignment::isParallelToAlignmentAxisForChild const): Deleted.
(WebCore::GridBaselineAlignment::baselineGroupForChild const): Deleted.
(WebCore::GridBaselineAlignment::baselineOffsetForChild const): Deleted.
* Source/WebCore/rendering/GridBaselineAlignment.h:
* Source/WebCore/rendering/GridLayoutFunctions.cpp:
(WebCore::GridLayoutFunctions::marginStartIsAuto):
(WebCore::GridLayoutFunctions::marginEndIsAuto):
(WebCore::GridLayoutFunctions::gridItemHasMargin):
(WebCore::GridLayoutFunctions::computeMarginLogicalSizeForGridItem):
(WebCore::GridLayoutFunctions::hasRelativeOrIntrinsicSizeForGridItem):
(WebCore::GridLayoutFunctions::extraMarginForSubgrid):
(WebCore::GridLayoutFunctions::extraMarginForSubgridAncestors):
(WebCore::GridLayoutFunctions::marginLogicalSizeForGridItem):
(WebCore::GridLayoutFunctions::isOrthogonalGridItem):
(WebCore::GridLayoutFunctions::isAspectRatioBlockSizeDependentGridItem):
(WebCore::GridLayoutFunctions::flowAwareDirectionForGridItem):
(WebCore::GridLayoutFunctions::overridingContainingBlockContentSizeForGridItem):
(WebCore::GridLayoutFunctions::isSubgridReversedDirection):
(WebCore::GridLayoutFunctions::childHasMargin): Deleted.
(WebCore::GridLayoutFunctions::computeMarginLogicalSizeForChild): Deleted.
(WebCore::GridLayoutFunctions::hasRelativeOrIntrinsicSizeForChild): Deleted.
(WebCore::GridLayoutFunctions::marginLogicalSizeForChild): Deleted.
(WebCore::GridLayoutFunctions::isOrthogonalChild): Deleted.
(WebCore::GridLayoutFunctions::isAspectRatioBlockSizeDependentChild): Deleted.
(WebCore::GridLayoutFunctions::flowAwareDirectionForChild): Deleted.
(WebCore::GridLayoutFunctions::overridingContainingBlockContentSizeForChild): Deleted.
* Source/WebCore/rendering/GridLayoutFunctions.h:
* Source/WebCore/rendering/GridMasonryLayout.cpp:
(WebCore::GridMasonryLayout::collectMasonryItems):
(WebCore::GridMasonryLayout::placeItemsUsingOrderModifiedDocumentOrder):
(WebCore::GridMasonryLayout::gridAreaForDefiniteGridAxisItem const):
(WebCore::GridMasonryLayout::setItemGridAxisContainingBlockToGridArea):
(WebCore::GridMasonryLayout::insertIntoGridAndLayoutItem):
(WebCore::GridMasonryLayout::masonryAxisMarginBoxForItem):
(WebCore::GridMasonryLayout::updateRunningPositions):
(WebCore::GridMasonryLayout::updateItemOffset):
(WebCore::GridMasonryLayout::offsetForGridItem const):
(WebCore::GridMasonryLayout::hasDefiniteGridAxisPosition const):
(WebCore::GridMasonryLayout::offsetForChild const): Deleted.
* Source/WebCore/rendering/GridMasonryLayout.h:
* Source/WebCore/rendering/GridTrackSizingAlgorithm.cpp:
(WebCore::hasRelativeMarginOrPaddingForGridItem):
(WebCore::hasRelativeOrIntrinsicSizeForGridItem):
(WebCore::shouldClearOverridingContainingBlockContentSizeForGridItem):
(WebCore::setOverridingContainingBlockContentSizeForGridItem):
(WebCore::GridTrackSizingAlgorithm::sizeTrackToFitNonSpanningItem):
(WebCore::GridTrackSizingAlgorithm::itemSizeForTrackSizeComputationPhase const):
(WebCore::GridTrackSizingAlgorithm::estimatedGridAreaBreadthForGridItem const):
(WebCore::GridTrackSizingAlgorithm::gridAreaBreadthForGridItem const):
(WebCore::GridTrackSizingAlgorithm::isIntrinsicSizedGridArea const):
(WebCore::GridTrackSizingAlgorithmStrategy::logicalHeightForGridItem const):
(WebCore::GridTrackSizingAlgorithmStrategy::minContentForGridItem const):
(WebCore::GridTrackSizingAlgorithmStrategy::maxContentForGridItem const):
(WebCore::GridTrackSizingAlgorithmStrategy::minSizeForGridItem const):
(WebCore::GridTrackSizingAlgorithm::canParticipateInBaselineAlignment const):
(WebCore::GridTrackSizingAlgorithm::participateInBaselineAlignment const):
(WebCore::GridTrackSizingAlgorithm::updateBaselineAlignmentContext):
(WebCore::GridTrackSizingAlgorithm::baselineOffsetForGridItem const):
(WebCore::GridTrackSizingAlgorithm::cacheBaselineAlignedItem):
(WebCore::GridTrackSizingAlgorithmStrategy::updateOverridingContainingBlockContentSizeForGridItem const):
(WebCore::GridTrackSizingAlgorithmStrategy::minLogicalSizeForGridItem const):
(WebCore::IndefiniteSizeStrategy::layoutGridItemForMinSizeComputation const):
(WebCore::IndefiniteSizeStrategy::accumulateFlexFraction const):
(WebCore::DefiniteSizeStrategy::minLogicalSizeForGridItem const):
(WebCore::DefiniteSizeStrategy::layoutGridItemForMinSizeComputation const):
(WebCore::DefiniteSizeStrategy::minContentForGridItem const):
(WebCore::computeSubgridMarginBorderPadding):
(WebCore::extraMarginFromSubgridAncestorGutters):
(WebCore::GridTrackSizingAlgorithm::shouldExcludeGridItemForMasonryTrackSizing const):
(WebCore::GridTrackSizingAlgorithm::accumulateIntrinsicSizesForTrack):
(WebCore::GridTrackSizingAlgorithm::accumulateIntrinsicSizesForTrackMasonry):
(WebCore::GridTrackSizingAlgorithm::computeIndefiniteItemsForMasonry const):
(WebCore::GridTrackSizingAlgorithm::setup):
(WebCore::GridTrackSizingAlgorithm::computeBaselineAlignmentContext):
(WebCore::GridTrackSizingAlgorithm::copyUsedTrackSizesForSubgrid):
(WebCore::hasRelativeMarginOrPaddingForChild): Deleted.
(WebCore::hasRelativeOrIntrinsicSizeForChild): Deleted.
(WebCore::shouldClearOverridingContainingBlockContentSizeForChild): Deleted.
(WebCore::setOverridingContainingBlockContentSizeForChild): Deleted.
(WebCore::GridTrackSizingAlgorithm::estimatedGridAreaBreadthForChild const): Deleted.
(WebCore::GridTrackSizingAlgorithm::gridAreaBreadthForChild const): Deleted.
(WebCore::GridTrackSizingAlgorithmStrategy::logicalHeightForChild const): Deleted.
(WebCore::GridTrackSizingAlgorithmStrategy::minContentForChild const): Deleted.
(WebCore::GridTrackSizingAlgorithmStrategy::maxContentForChild const): Deleted.
(WebCore::GridTrackSizingAlgorithmStrategy::minSizeForChild const): Deleted.
(WebCore::GridTrackSizingAlgorithm::baselineOffsetForChild const): Deleted.
(WebCore::GridTrackSizingAlgorithmStrategy::updateOverridingContainingBlockContentSizeForChild const): Deleted.
(WebCore::GridTrackSizingAlgorithmStrategy::minLogicalSizeForChild const): Deleted.
(WebCore::DefiniteSizeStrategy::minLogicalSizeForChild const): Deleted.
(WebCore::DefiniteSizeStrategy::minContentForChild const): Deleted.
* Source/WebCore/rendering/GridTrackSizingAlgorithm.h:
* Source/WebCore/rendering/RenderGrid.cpp:
(WebCore::RenderGrid::selfAlignmentForGridItem const):
(WebCore::RenderGrid::selfAlignmentChangedToStretch const):
(WebCore::RenderGrid::selfAlignmentChangedFromStretch const):
(WebCore::RenderGrid::styleDidChange):
(WebCore::RenderGrid::repeatTracksSizingIfNeeded):
(WebCore::cacheBaselineAlignedGridItems):
(WebCore::RenderGrid::canSetColumnAxisStretchRequirementForItem const):
(WebCore::RenderGrid::computeIntrinsicLogicalWidths const):
(WebCore::insertIntoGrid):
(WebCore::RenderGrid::placeItemsOnGrid):
(WebCore::RenderGrid::gridItemsLayoutRects):
(WebCore::RenderGrid::performPreLayoutForGridItems const):
(WebCore::RenderGrid::populateExplicitGridAndOrderIterator):
(WebCore::overrideSizeChanged):
(WebCore::hasRelativeBlockAxisSize):
(WebCore::RenderGrid::updateGridAreaLogicalSize const):
(WebCore::RenderGrid::updateGridAreaForAspectRatioItems):
(WebCore::RenderGrid::layoutGridItems):
(WebCore::RenderGrid::layoutMasonryItems):
(WebCore::RenderGrid::prepareGridItemForPositionedLayout):
(WebCore::RenderGrid::hasStaticPositionForGridItem const):
(WebCore::RenderGrid::layoutPositionedObject):
(WebCore::RenderGrid::gridAreaBreadthForGridItemIncludingAlignmentOffsets const):
(WebCore::computeOverflowAlignmentOffset):
(WebCore::RenderGrid::availableAlignmentSpaceForGridItemBeforeStretching const):
(WebCore::RenderGrid::alignSelfForGridItem const):
(WebCore::RenderGrid::justifySelfForGridItem const):
(WebCore::RenderGrid::aspectRatioPrefersInline):
(WebCore::RenderGrid::allowedToStretchGridItemAlongColumnAxis const):
(WebCore::RenderGrid::allowedToStretchGridItemAlongRowAxis const):
(WebCore::RenderGrid::applyStretchAlignmentToGridItemIfNeeded):
(WebCore::RenderGrid::applySubgridStretchAlignmentToGridItemIfNeeded):
(WebCore::RenderGrid::hasAutoMarginsInColumnAxis const):
(WebCore::RenderGrid::hasAutoMarginsInRowAxis const):
(WebCore::RenderGrid::updateAutoMarginsInRowAxisIfNeeded):
(WebCore::RenderGrid::updateAutoMarginsInColumnAxisIfNeeded):
(WebCore::RenderGrid::isChildEligibleForMarginTrim const):
(WebCore::RenderGrid::isBaselineAlignmentForGridItem const):
(WebCore::RenderGrid::firstLineBaseline const):
(WebCore::RenderGrid::lastLineBaseline const):
(WebCore::RenderGrid::getBaselineGridItem const):
(WebCore::RenderGrid::columnAxisBaselineOffsetForGridItem const):
(WebCore::RenderGrid::rowAxisBaselineOffsetForGridItem const):
(WebCore::RenderGrid::columnAxisPositionForGridItem const):
(WebCore::RenderGrid::rowAxisPositionForGridItem const):
(WebCore::RenderGrid::columnAxisOffsetForGridItem const):
(WebCore::RenderGrid::rowAxisOffsetForGridItem const):
(WebCore::RenderGrid::isSubgridInParentDirection const):
(WebCore::RenderGrid::gridAreaBreadthForOutOfFlowGridItem):
(WebCore::RenderGrid::logicalOffsetForOutOfFlowGridItem const):
(WebCore::RenderGrid::gridAreaPositionForOutOfFlowGridItem const):
(WebCore::RenderGrid::gridAreaPositionForInFlowGridItem const):
(WebCore::RenderGrid::gridAreaPositionForGridItem const):
(WebCore::RenderGrid::setLogicalPositionForGridItem const):
(WebCore::RenderGrid::setLogicalOffsetForGridItem const):
(WebCore::RenderGrid::logicalOffsetForGridItem const):
(WebCore::RenderGrid::numTracks const):
(WebCore::RenderGrid::paintChildren):
(WebCore::RenderGrid::hasAutoSizeInColumnAxis const):
(WebCore::RenderGrid::hasAutoSizeInRowAxis const):
(WebCore::RenderGrid::computeGridPositionsForOutOfFlowGridItem const):
(WebCore::RenderGrid::gridSpanForOutOfFlowGridItem const):
(WebCore::RenderGrid::gridSpanForGridItem const):
(WebCore::RenderGrid::selfAlignmentForChild const): Deleted.
(WebCore::cacheBaselineAlignedChildren): Deleted.
(WebCore::RenderGrid::prepareChildForPositionedLayout): Deleted.
(WebCore::RenderGrid::hasStaticPositionForChild const): Deleted.
(WebCore::RenderGrid::gridAreaBreadthForChildIncludingAlignmentOffsets const): Deleted.
(WebCore::RenderGrid::availableAlignmentSpaceForChildBeforeStretching const): Deleted.
(WebCore::RenderGrid::alignSelfForChild const): Deleted.
(WebCore::RenderGrid::justifySelfForChild const): Deleted.
(WebCore::RenderGrid::allowedToStretchChildAlongColumnAxis const): Deleted.
(WebCore::RenderGrid::allowedToStretchChildAlongRowAxis const): Deleted.
(WebCore::RenderGrid::applyStretchAlignmentToChildIfNeeded): Deleted.
(WebCore::RenderGrid::applySubgridStretchAlignmentToChildIfNeeded): Deleted.
(WebCore::RenderGrid::isBaselineAlignmentForChild const): Deleted.
(WebCore::RenderGrid::getBaselineChild const): Deleted.
(WebCore::RenderGrid::columnAxisBaselineOffsetForChild const): Deleted.
(WebCore::RenderGrid::rowAxisBaselineOffsetForChild const): Deleted.
(WebCore::RenderGrid::columnAxisPositionForChild const): Deleted.
(WebCore::RenderGrid::rowAxisPositionForChild const): Deleted.
(WebCore::RenderGrid::columnAxisOffsetForChild const): Deleted.
(WebCore::RenderGrid::rowAxisOffsetForChild const): Deleted.
(WebCore::RenderGrid::gridAreaBreadthForOutOfFlowChild): Deleted.
(WebCore::RenderGrid::logicalOffsetForOutOfFlowChild const): Deleted.
(WebCore::RenderGrid::gridAreaPositionForOutOfFlowChild const): Deleted.
(WebCore::RenderGrid::gridAreaPositionForInFlowChild const): Deleted.
(WebCore::RenderGrid::gridAreaPositionForChild const): Deleted.
(WebCore::RenderGrid::setLogicalPositionForChild const): Deleted.
(WebCore::RenderGrid::setLogicalOffsetForChild const): Deleted.
(WebCore::RenderGrid::logicalOffsetForChild const): Deleted.
(WebCore::RenderGrid::computeGridPositionsForOutOfFlowChild const): Deleted.
(WebCore::RenderGrid::gridSpanForOutOfFlowChild const): Deleted.
(WebCore::RenderGrid::gridSpanForChild const): Deleted.
* Source/WebCore/rendering/RenderGrid.h:
* Source/WebCore/rendering/style/GridPositionsResolver.cpp:
(WebCore::NamedLineCollection::NamedLineCollection):
(WebCore::GridPositionsResolver::explicitGridColumnCount):
(WebCore::GridPositionsResolver::explicitGridRowCount):

Canonical link: <a href="https://commits.webkit.org/281294@main">https://commits.webkit.org/281294@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/e46d7af4b1c0a0635d1f71c221dd836c6df52a88

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/59370 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/38713 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/11890 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/63284 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/9813 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/61499 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/46367 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/10044 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/48199 "Passed tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/6964 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/61400 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/36159 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/51386 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/29024 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/32869 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/8641 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/8817 "Built successfully") | 
| | [❌ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/54824 "2 api tests failed or timed out") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/8921 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/65017 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/3298 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/8837 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/55546 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/3309 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/51383 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/55652 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/13160 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/2739 "Passed tests") | | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/34529 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/35612 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/36698 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/35357 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

alanbaradlay commented on Source/WebCore/rendering/Grid.h:
```diff
@@ -136,7 +136,7 @@ class GridIterator {
     unsigned m_rowIndex;
     unsigned m_columnIndex;
-    unsigned m_childIndex;
+    unsigned m_GridItemIndex;
```
should read m_gridItemIndex

sammygill committed:
[Grid] Rename child/children references to gridItem/gridItems.
https://bugs.webkit.org/show_bug.cgi?id=276978
rdar://problem/132364014

Reviewed by Alan Baradlay.

These renderers participate in grid layout and are grid items so
renaming them to such in the code makes it more consistent with the
spec and easier to read. There are some instances that are used as part
of generic render tree logic and not specifially grid layout which I
left alone.

There were also a couple of instances of "childIterator" in the
grid track sizing algorithm that I changed to "subgridIterator,"" since
that's what they are.

* Source/WebCore/rendering/AncestorSubgridIterator.cpp:
(WebCore::AncestorSubgridIterator::operator++):
* Source/WebCore/rendering/Grid.cpp:
(WebCore::Grid::insert):
(WebCore::GridIterator::GridIterator):
(WebCore::GridIterator::nextGridItem):
(WebCore::GridIterator::isEmptyAreaEnough const):
(WebCore::GridIterator::createForSubgrid):
* Source/WebCore/rendering/Grid.h:
* Source/WebCore/rendering/GridBaselineAlignment.cpp:
(WebCore::GridBaselineAlignment::logicalAscentForGridItem const):
(WebCore::GridBaselineAlignment::ascentForGridItem const):
(WebCore::GridBaselineAlignment::descentForGridItem const):
(WebCore::GridBaselineAlignment::isDescentBaselineForGridItem const):
(WebCore::GridBaselineAlignment::isOrthogonalGridItemForBaseline const):
(WebCore::GridBaselineAlignment::isParallelToAlignmentAxisForGridItem const):
(WebCore::GridBaselineAlignment::baselineGroupForGridItem const):
(WebCore::GridBaselineAlignment::updateBaselineAlignmentContext):
(WebCore::GridBaselineAlignment::baselineOffsetForGridItem const):
(WebCore::GridBaselineAlignment::logicalAscentForChild const): Deleted.
(WebCore::GridBaselineAlignment::ascentForChild const): Deleted.
(WebCore::GridBaselineAlignment::descentForChild const): Deleted.
(WebCore::GridBaselineAlignment::isDescentBaselineForChild const): Deleted.
(WebCore::GridBaselineAlignment::isOrthogonalChildForBaseline const): Deleted.
(WebCore::GridBaselineAlignment::isParallelToAlignmentAxisForChild const): Deleted.
(WebCore::GridBaselineAlignment::baselineGroupForChild const): Deleted.
(WebCore::GridBaselineAlignment::baselineOffsetForChild const): Deleted.
* Source/WebCore/rendering/GridBaselineAlignment.h:
* Source/WebCore/rendering/GridLayoutFunctions.cpp:
(WebCore::GridLayoutFunctions::marginStartIsAuto):
(WebCore::GridLayoutFunctions::marginEndIsAuto):
(WebCore::GridLayoutFunctions::gridItemHasMargin):
(WebCore::GridLayoutFunctions::computeMarginLogicalSizeForGridItem):
(WebCore::GridLayoutFunctions::hasRelativeOrIntrinsicSizeForGridItem):
(WebCore::GridLayoutFunctions::extraMarginForSubgrid):
(WebCore::GridLayoutFunctions::extraMarginForSubgridAncestors):
(WebCore::GridLayoutFunctions::marginLogicalSizeForGridItem):
(WebCore::GridLayoutFunctions::isOrthogonalGridItem):
(WebCore::GridLayoutFunctions::isAspectRatioBlockSizeDependentGridItem):
(WebCore::GridLayoutFunctions::flowAwareDirectionForGridItem):
(WebCore::GridLayoutFunctions::overridingContainingBlockContentSizeForGridItem):
(WebCore::GridLayoutFunctions::isSubgridReversedDirection):
(WebCore::GridLayoutFunctions::childHasMargin): Deleted.
(WebCore::GridLayoutFunctions::computeMarginLogicalSizeForChild): Deleted.
(WebCore::GridLayoutFunctions::hasRelativeOrIntrinsicSizeForChild): Deleted.
(WebCore::GridLayoutFunctions::marginLogicalSizeForChild): Deleted.
(WebCore::GridLayoutFunctions::isOrthogonalChild): Deleted.
(WebCore::GridLayoutFunctions::isAspectRatioBlockSizeDependentChild): Deleted.
(WebCore::GridLayoutFunctions::flowAwareDirectionForChild): Deleted.
(WebCore::GridLayoutFunctions::overridingContainingBlockContentSizeForChild): Deleted.
* Source/WebCore/rendering/GridLayoutFunctions.h:
* Source/WebCore/rendering/GridMasonryLayout.cpp:
(WebCore::GridMasonryLayout::collectMasonryItems):
(WebCore::GridMasonryLayout::placeItemsUsingOrderModifiedDocumentOrder):
(WebCore::GridMasonryLayout::gridAreaForDefiniteGridAxisItem const):
(WebCore::GridMasonryLayout::setItemGridAxisContainingBlockToGridArea):
(WebCore::GridMasonryLayout::insertIntoGridAndLayoutItem):
(WebCore::GridMasonryLayout::masonryAxisMarginBoxForItem):
(WebCore::GridMasonryLayout::updateRunningPositions):
(WebCore::GridMasonryLayout::updateItemOffset):
(WebCore::GridMasonryLayout::offsetForGridItem const):
(WebCore::GridMasonryLayout::hasDefiniteGridAxisPosition const):
(WebCore::GridMasonryLayout::offsetForChild const): Deleted.
* Source/WebCore/rendering/GridMasonryLayout.h:
* Source/WebCore/rendering/GridTrackSizingAlgorithm.cpp:
(WebCore::hasRelativeMarginOrPaddingForGridItem):
(WebCore::hasRelativeOrIntrinsicSizeForGridItem):
(WebCore::shouldClearOverridingContainingBlockContentSizeForGridItem):
(WebCore::setOverridingContainingBlockContentSizeForGridItem):
(WebCore::GridTrackSizingAlgorithm::sizeTrackToFitNonSpanningItem):
(WebCore::GridTrackSizingAlgorithm::itemSizeForTrackSizeComputationPhase const):
(WebCore::GridTrackSizingAlgorithm::estimatedGridAreaBreadthForGridItem const):
(WebCore::GridTrackSizingAlgorithm::gridAreaBreadthForGridItem const):
(WebCore::GridTrackSizingAlgorithm::isIntrinsicSizedGridArea const):
(WebCore::GridTrackSizingAlgorithmStrategy::logicalHeightForGridItem const):
(WebCore::GridTrackSizingAlgorithmStrategy::minContentForGridItem const):
(WebCore::GridTrackSizingAlgorithmStrategy::maxContentForGridItem const):
(WebCore::GridTrackSizingAlgorithmStrategy::minSizeForGridItem const):
(WebCore::GridTrackSizingAlgorithm::canParticipateInBaselineAlignment const):
(WebCore::GridTrackSizingAlgorithm::participateInBaselineAlignment const):
(WebCore::GridTrackSizingAlgorithm::updateBaselineAlignmentContext):
(WebCore::GridTrackSizingAlgorithm::baselineOffsetForGridItem const):
(WebCore::GridTrackSizingAlgorithm::cacheBaselineAlignedItem):
(WebCore::GridTrackSizingAlgorithmStrategy::updateOverridingContainingBlockContentSizeForGridItem const):
(WebCore::GridTrackSizingAlgorithmStrategy::minLogicalSizeForGridItem const):
(WebCore::IndefiniteSizeStrategy::layoutGridItemForMinSizeComputation const):
(WebCore::IndefiniteSizeStrategy::accumulateFlexFraction const):
(WebCore::DefiniteSizeStrategy::minLogicalSizeForGridItem const):
(WebCore::DefiniteSizeStrategy::layoutGridItemForMinSizeComputation const):
(WebCore::DefiniteSizeStrategy::minContentForGridItem const):
(WebCore::computeSubgridMarginBorderPadding):
(WebCore::extraMarginFromSubgridAncestorGutters):
(WebCore::GridTrackSizingAlgorithm::shouldExcludeGridItemForMasonryTrackSizing const):
(WebCore::GridTrackSizingAlgorithm::accumulateIntrinsicSizesForTrack):
(WebCore::GridTrackSizingAlgorithm::accumulateIntrinsicSizesForTrackMasonry):
(WebCore::GridTrackSizingAlgorithm::computeIndefiniteItemsForMasonry const):
(WebCore::GridTrackSizingAlgorithm::setup):
(WebCore::GridTrackSizingAlgorithm::computeBaselineAlignmentContext):
(WebCore::GridTrackSizingAlgorithm::copyUsedTrackSizesForSubgrid):
(WebCore::hasRelativeMarginOrPaddingForChild): Deleted.
(WebCore::hasRelativeOrIntrinsicSizeForChild): Deleted.
(WebCore::shouldClearOverridingContainingBlockContentSizeForChild): Deleted.
(WebCore::setOverridingContainingBlockContentSizeForChild): Deleted.
(WebCore::GridTrackSizingAlgorithm::estimatedGridAreaBreadthForChild const): Deleted.
(WebCore::GridTrackSizingAlgorithm::gridAreaBreadthForChild const): Deleted.
(WebCore::GridTrackSizingAlgorithmStrategy::logicalHeightForChild const): Deleted.
(WebCore::GridTrackSizingAlgorithmStrategy::minContentForChild const): Deleted.
(WebCore::GridTrackSizingAlgorithmStrategy::maxContentForChild const): Deleted.
(WebCore::GridTrackSizingAlgorithmStrategy::minSizeForChild const): Deleted.
(WebCore::GridTrackSizingAlgorithm::baselineOffsetForChild const): Deleted.
(WebCore::GridTrackSizingAlgorithmStrategy::updateOverridingContainingBlockContentSizeForChild const): Deleted.
(WebCore::GridTrackSizingAlgorithmStrategy::minLogicalSizeForChild const): Deleted.
(WebCore::DefiniteSizeStrategy::minLogicalSizeForChild const): Deleted.
(WebCore::DefiniteSizeStrategy::minContentForChild const): Deleted.
* Source/WebCore/rendering/GridTrackSizingAlgorithm.h:
* Source/WebCore/rendering/RenderGrid.cpp:
(WebCore::RenderGrid::selfAlignmentForGridItem const):
(WebCore::RenderGrid::selfAlignmentChangedToStretch const):
(WebCore::RenderGrid::selfAlignmentChangedFromStretch const):
(WebCore::RenderGrid::styleDidChange):
(WebCore::RenderGrid::repeatTracksSizingIfNeeded):
(WebCore::cacheBaselineAlignedGridItems):
(WebCore::RenderGrid::canSetColumnAxisStretchRequirementForItem const):
(WebCore::RenderGrid::computeIntrinsicLogicalWidths const):
(WebCore::insertIntoGrid):
(WebCore::RenderGrid::placeItemsOnGrid):
(WebCore::RenderGrid::gridItemsLayoutRects):
(WebCore::RenderGrid::performPreLayoutForGridItems const):
(WebCore::RenderGrid::populateExplicitGridAndOrderIterator):
(WebCore::overrideSizeChanged):
(WebCore::hasRelativeBlockAxisSize):
(WebCore::RenderGrid::updateGridAreaLogicalSize const):
(WebCore::RenderGrid::updateGridAreaForAspectRatioItems):
(WebCore::RenderGrid::layoutGridItems):
(WebCore::RenderGrid::layoutMasonryItems):
(WebCore::RenderGrid::prepareGridItemForPositionedLayout):
(WebCore::RenderGrid::hasStaticPositionForGridItem const):
(WebCore::RenderGrid::layoutPositionedObject):
(WebCore::RenderGrid::gridAreaBreadthForGridItemIncludingAlignmentOffsets const):
(WebCore::computeOverflowAlignmentOffset):
(WebCore::RenderGrid::availableAlignmentSpaceForGridItemBeforeStretching const):
(WebCore::RenderGrid::alignSelfForGridItem const):
(WebCore::RenderGrid::justifySelfForGridItem const):
(WebCore::RenderGrid::aspectRatioPrefersInline):
(WebCore::RenderGrid::allowedToStretchGridItemAlongColumnAxis const):
(WebCore::RenderGrid::allowedToStretchGridItemAlongRowAxis const):
(WebCore::RenderGrid::applyStretchAlignmentToGridItemIfNeeded):
(WebCore::RenderGrid::applySubgridStretchAlignmentToGridItemIfNeeded):
(WebCore::RenderGrid::hasAutoMarginsInColumnAxis const):
(WebCore::RenderGrid::hasAutoMarginsInRowAxis const):
(WebCore::RenderGrid::updateAutoMarginsInRowAxisIfNeeded):
(WebCore::RenderGrid::updateAutoMarginsInColumnAxisIfNeeded):
(WebCore::RenderGrid::isChildEligibleForMarginTrim const):
(WebCore::RenderGrid::isBaselineAlignmentForGridItem const):
(WebCore::RenderGrid::firstLineBaseline const):
(WebCore::RenderGrid::lastLineBaseline const):
(WebCore::RenderGrid::getBaselineGridItem const):
(WebCore::RenderGrid::columnAxisBaselineOffsetForGridItem const):
(WebCore::RenderGrid::rowAxisBaselineOffsetForGridItem const):
(WebCore::RenderGrid::columnAxisPositionForGridItem const):
(WebCore::RenderGrid::rowAxisPositionForGridItem const):
(WebCore::RenderGrid::columnAxisOffsetForGridItem const):
(WebCore::RenderGrid::rowAxisOffsetForGridItem const):
(WebCore::RenderGrid::isSubgridInParentDirection const):
(WebCore::RenderGrid::gridAreaBreadthForOutOfFlowGridItem):
(WebCore::RenderGrid::logicalOffsetForOutOfFlowGridItem const):
(WebCore::RenderGrid::gridAreaPositionForOutOfFlowGridItem const):
(WebCore::RenderGrid::gridAreaPositionForInFlowGridItem const):
(WebCore::RenderGrid::gridAreaPositionForGridItem const):
(WebCore::RenderGrid::setLogicalPositionForGridItem const):
(WebCore::RenderGrid::setLogicalOffsetForGridItem const):
(WebCore::RenderGrid::logicalOffsetForGridItem const):
(WebCore::RenderGrid::numTracks const):
(WebCore::RenderGrid::paintChildren):
(WebCore::RenderGrid::hasAutoSizeInColumnAxis const):
(WebCore::RenderGrid::hasAutoSizeInRowAxis const):
(WebCore::RenderGrid::computeGridPositionsForOutOfFlowGridItem const):
(WebCore::RenderGrid::gridSpanForOutOfFlowGridItem const):
(WebCore::RenderGrid::gridSpanForGridItem const):
(WebCore::RenderGrid::selfAlignmentForChild const): Deleted.
(WebCore::cacheBaselineAlignedChildren): Deleted.
(WebCore::RenderGrid::prepareChildForPositionedLayout): Deleted.
(WebCore::RenderGrid::hasStaticPositionForChild const): Deleted.
(WebCore::RenderGrid::gridAreaBreadthForChildIncludingAlignmentOffsets const): Deleted.
(WebCore::RenderGrid::availableAlignmentSpaceForChildBeforeStretching const): Deleted.
(WebCore::RenderGrid::alignSelfForChild const): Deleted.
(WebCore::RenderGrid::justifySelfForChild const): Deleted.
(WebCore::RenderGrid::allowedToStretchChildAlongColumnAxis const): Deleted.
(WebCore::RenderGrid::allowedToStretchChildAlongRowAxis const): Deleted.
(WebCore::RenderGrid::applyStretchAlignmentToChildIfNeeded): Deleted.
(WebCore::RenderGrid::applySubgridStretchAlignmentToChildIfNeeded): Deleted.
(WebCore::RenderGrid::isBaselineAlignmentForChild const): Deleted.
(WebCore::RenderGrid::getBaselineChild const): Deleted.
(WebCore::RenderGrid::columnAxisBaselineOffsetForChild const): Deleted.
(WebCore::RenderGrid::rowAxisBaselineOffsetForChild const): Deleted.
(WebCore::RenderGrid::columnAxisPositionForChild const): Deleted.
(WebCore::RenderGrid::rowAxisPositionForChild const): Deleted.
(WebCore::RenderGrid::columnAxisOffsetForChild const): Deleted.
(WebCore::RenderGrid::rowAxisOffsetForChild const): Deleted.
(WebCore::RenderGrid::gridAreaBreadthForOutOfFlowChild): Deleted.
(WebCore::RenderGrid::logicalOffsetForOutOfFlowChild const): Deleted.
(WebCore::RenderGrid::gridAreaPositionForOutOfFlowChild const): Deleted.
(WebCore::RenderGrid::gridAreaPositionForInFlowChild const): Deleted.
(WebCore::RenderGrid::gridAreaPositionForChild const): Deleted.
(WebCore::RenderGrid::setLogicalPositionForChild const): Deleted.
(WebCore::RenderGrid::setLogicalOffsetForChild const): Deleted.
(WebCore::RenderGrid::logicalOffsetForChild const): Deleted.
(WebCore::RenderGrid::computeGridPositionsForOutOfFlowChild const): Deleted.
(WebCore::RenderGrid::gridSpanForOutOfFlowChild const): Deleted.
(WebCore::RenderGrid::gridSpanForChild const): Deleted.
* Source/WebCore/rendering/RenderGrid.h:
* Source/WebCore/rendering/style/GridPositionsResolver.cpp:
(WebCore::NamedLineCollection::NamedLineCollection):
(WebCore::GridPositionsResolver::explicitGridColumnCount):
(WebCore::GridPositionsResolver::explicitGridRowCount):

Canonical link: https://commits.webkit.org/281294@main
